compiler/compiler/src/compiler.rs

```diff
@@ -233,13 +236,19 @@ impl<'a> Compiler<'a> {
     }
 
     /// Writes the AST to a JSON file.
-    fn write_ast_to_json(&self, file_name: &str) -> Result<()> {
+    fn write_ast_to_json(&self, file_suffix: &str) -> Result<()> {
         // Remove `Span`s if they are not enabled.
         if self.output_options.spans_enabled {
-            self.ast.to_json_file(self.output_directory.clone(), file_name)?;
+            self.ast.to_json_file(
+                self.output_directory.clone(),
+                &format!("{}.{file_suffix}", self.program_name),
+            )?;
         } else {
-            self.ast
-                .to_json_file_without_keys(self.output_directory.clone(), file_name, &["span"])?;
+            self.ast.to_json_file_without_keys(
+                self.output_directory.clone(),
+                &format!("{}.{file_suffix}", self.program_name),
```

Comment:
I may be misunderstanding the code, but I thought that currently the network name is `aleo`, not `leo` (which is the file extension)?

Reply:
The code just prepends the program name, e.g `foo` in `foo.aleo` to ensure uniqueness for the output files.
The network name is `aleo` but has nothing to do with this code.